docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,20 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed| |userbreak|
+         - TimeLock Server, if configured to use the async lock service, will now throw if a client attempts to start a transaction via the sync lock service.
+           Previously, users which have clients (for the same namespace) running both pre- and post-0.49.0 versions of AtlasDB were able to run transactions against the sync and async lock services concurrently, thus ignoring write-write conflict checks.
```

Comment:
nit: replace `ignoring write-write conflict checks` with `breaking the guarantees of the lock service`